src/wallet/wallet.h

```diff
@@ -88,6 +88,10 @@ enum WalletFeature
     FEATURE_COMPRPUBKEY = 60000, // compressed public keys
 
     FEATURE_HD = 130000, // Hierarchical key derivation after BIP32 (HD Wallet)
+
+    //TODO: FEATURE_HD_SPLIT needs to be bumped to 140000 once we branch of 0.14 //
```

Comment:
I think if we branch of 0.14 we would need to bump it to `1499000`. If someone was using this patch in production... hes.. in serious troubles anyways. :)